.travis.yml

```diff
@@ -56,27 +56,20 @@ env:
     - >-
       BUILD_TYPE=centaurHoricromtalEngineUpgradePapiV2alpha1
       BUILD_MARIADB=10.3
-    - >-
-      BUILD_TYPE=centaurPapiUpgradePapiV1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradePapiV2alpha1
-      BUILD_MYSQL=5.7
-    - >-
-      BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV2alpha1
-      BUILD_MYSQL=5.7
+    # This is not currently running any tests, see BT-81.
+    # - >-
+    #  BUILD_TYPE=centaurPapiUpgradePapiV2alpha1
+    #  BUILD_MYSQL=5.7
+    # This is not currently running any tests, see BT-81.
+    # - >-
+    #  BUILD_TYPE=centaurPapiUpgradeNewWorkflowsPapiV2alpha1
+    #  BUILD_MYSQL=5.7
     - >-
       BUILD_TYPE=centaurLocal
       BUILD_MYSQL=5.7
     - >-
       BUILD_TYPE=centaurLocal
       BUILD_POSTGRESQL=11.3
-    - >-
-      BUILD_TYPE=centaurPapiV1
-      BUILD_MYSQL=5.7
```

Comment:
As of 6e977e6bd57ea80e3eef86c98228524151ce8061, this shell script is still in the commit.

Reply:
oops